ui/ducks/send/send.js

```diff
@@ -2260,9 +2260,9 @@ export function resetRecipientInput() {
     const state = getState();
     const chainId = getCurrentChainId(state);
     await dispatch(addHistoryEntry(`sendFlow - user cleared recipient input`));
+    await dispatch(resetDomainResolution());
```

Comment:
?

Reply:
Had to add this in to clear resolution when hitting the x to clear input in the send screen. Otherwise the resolution would hang.